installer/read_secrets.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash -e
 
 ENVIRONMENT=${1:?"Usage: read_secrets.sh ENVIRONMENT"}
+PREFIX=${VAULT_PATH:-secret/k8s_operator/$ENVIRONMENT}
 
 mkdir -p secrets
 
-COMPONENTS=`vault kv list --format=yaml secret/k8s_operator/$ENVIRONMENT | yq -r '.[]'`
+COMPONENTS=`vault kv list --format=yaml $VAULT_PATH | yq -r '.[]'`
 for SECRET in $COMPONENTS
 do
   if [ $SECRET != "efd/" ] && [ $SECRET != "ts/" ]; then
-    vault kv get --field=data --format=json secret/k8s_operator/$ENVIRONMENT/$SECRET > secrets/$SECRET
+    vault kv get --field=data --format=json $VAULT_PATH/$SECRET > secrets/$SECRET
```

Comment:
You're setting `PREFIX` but then using `VAULT_PATH`, which I don't believe will work.